python/docs/source/examples/index.md

```diff
@@ -2,11 +2,13 @@
 
 The [Time-Centric Calculations example](./time_centric.ipynb) shows how to work with time and produce past training examples and recent results for applying models.
 
-The [Bluesky Firehose example](./bluesky.md) show how to read and aggregate events from the Bluesky firehose.
+The [Bluesky Firehose example](./bluesky.md) shows how to read and aggregate messages from the Bluesky firehose.
+This demonstrates how to use Kaskada to connect in real-time and parse messages as part of the query.
```

Comment:
Thought it would be good to have a blurb about *why* the example is interesting rathre than just what it does.